include/xbt/string.hpp

```diff
@@ -7,7 +7,6 @@
 #ifndef SIMGRID_XBT_STRING_HPP
 #define SIMGRID_XBT_STRING_HPP
 
-#include "src/internal_config.h"
```

Comment:
In fact we don't need `internal_config.h` but we need `simgrid_config.h` for `HAVE_MC`.